pyproject.toml

```diff
@@ -89,6 +90,11 @@ docs = [
 ]
 fix = [ "pre-commit>=3.5" ]
 pkg-meta = [ "check-wheel-contents>=0.6.1", "twine>=6.1", "uv>=0.5.22" ]
+coverage = [
+  "covdefaults>=2.3",
```

Comment:
You can remove this for now, as we don't use it in this project, though we should add it at some point to the core test suite, because allows you to do conditional coverage markers 😊

Reply:
I'm interested, but I'll remove it for now!!